samples/system-test/quickstart.test.js

```diff
@@ -16,30 +16,24 @@
 'use strict';
 
 const path = require(`path`);
-const proxyquire = require(`proxyquire`).noPreserveCache();
-const sinon = require(`sinon`);
-const test = require(`ava`);
+const tools = require('@google-cloud/nodejs-repo-tools');
+const assert = require('assert');
 
 const cmd = `node quickstart.js`;
 const cwd = path.join(__dirname, `..`);
 
-test(`should make some API calls`, (t) => {
-  const docMock = {
-    set: sinon.stub().returns(Promise.resolve()),
-    update: sinon.stub().returns(Promise.resolve()),
-    get: sinon.stub().returns(Promise.resolve()),
-    delete: sinon.stub().returns(Promise.resolve()),
-  };
-
-  function FirestoreMock() {}
-  FirestoreMock.prototype.doc = sinon.stub().returns(docMock);
-
-  proxyquire(`../quickstart`, {
-    '@google-cloud/firestore': FirestoreMock,
-  });
-
-  t.is(docMock.set.callCount, 1);
-  t.is(docMock.update.callCount, 1);
-  t.is(docMock.get.callCount, 1);
-  t.is(docMock.delete.callCount, 1);
-});
+
+describe('should make some API calls',function(){
```

Comment:
Lets use arrow functions instead of `function(){`

Reply:
Done.